e2e/runtime_test.go

```diff
@@ -19,26 +19,30 @@ import (
 func TestRuntime(t *testing.T) {
 	limavm.Require(t)
 
-	dirMountedInVM := t.TempDir()
-
-	rootpathPrefix := filepath.Join(dirMountedInVM, "fake-root")
-	runtimeBin, err := repo.BuildRuntimeBin(t.TempDir(), rootpathPrefix, limavm.BinBuildEnv)
+	rootpathPrefixInVM := filepath.Join("/tmp", "remoteproc-simulator-fake-root-for-standalone-runtime")
```

Comment:
While VMs are isolated, adding a unique suffix would:
  1. Prevent issues if VM cleanup fails
  2. Make debugging easier (correlate paths to specific test runs)
  3. Be more defensive against future changes
```suggestion
	rootpathPrefixInVM := filepath.Join("/tmp", fmt.Sprintf("remoteproc-sim-%s", testID(t)))
```
This pattern is repeated in a few places